Description: Refer to Digest::SHA instead of Digest::SHA1 as
 Digest::SHA is in perl core since 5.9.3.
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=74213
Forwarded: yes
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2012-01-21

--- a/lib/Digest/BubbleBabble.pm
+++ b/lib/Digest/BubbleBabble.pm
@@ -60,7 +60,7 @@
     my $fingerprint = bubblebabble( Digest => "Pineapple" );
 
     # Create a fingerprint of a SHA-1 hash.
-    use Digest::SHA1 qw( sha1 );
+    use Digest::SHA qw( sha1 );
     $fingerprint = bubblebabble( Digest => sha1("hello") );
 
 =head1 DESCRIPTION
@@ -77,7 +77,7 @@
 (and, consequently, by I<Net::SSH::Perl>, the Perl SSH
 implementation) to display easy-to-remember key fingerprints.
 The key (a DSA or RSA key) is converted into a textual form,
-digested using I<Digest::SHA1>, and run through I<bubblebabble>
+digested using I<Digest::SHA>, and run through I<bubblebabble>
 to create the key fingerprint.
 
 =head1 USAGE
@@ -93,7 +93,7 @@
 Currently takes only one pair of arguments, the key of
 which must be I<Digest>, the value of which is the actual
 message digest I<$digest>. You should generate this message
-digest yourself using either I<Digest::MD5> of I<Digest::SHA1>.
+digest yourself using either I<Digest::MD5> of I<Digest::SHA>.
 
 Returns the bubble babble form of the digest.
 
@@ -114,4 +114,4 @@
 Except where otherwise noted, Digest::BubbleBabble is Copyright
 2001 Benjamin Trott. All rights reserved.
 
-=cut
\ No newline at end of file
+=cut
